packages/sdk-metrics/src/Meter.ts

```diff
@@ -41,7 +41,6 @@ import {
   UpDownCounterInstrument,
 } from './Instruments';
 import { MeterSharedState } from './state/MeterSharedState';
-import { Gauge } from './types';
```

Comment:
We should keep using `Gauge` from `./types` as it could still happen that `@opentelemetry/api@1.3.0` which does not have the `Gauge` type.